client/src/components/friendModal/FriendModal.jsx

```diff
@@ -0,0 +1,153 @@
+import { Box, makeStyles, List, ListItem, Grid, Divider, Button, withStyles } from '@material-ui/core';
+import React, { useState } from 'react';
+import { useValue } from '../../utils/';
+import { Modal } from '../common/Modal/Modal';
+import { InputField } from '../common/InputField/InputField';
+import FriendItem from './FriendItem';
+
+const useStyles = makeStyles((theme) => ({
+  friendModal: {
+    display: 'flex',
+    width: 500,
```

Comment:
use theme value not raw value. otherwise it wont serve the purpose of theme provider